sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/workspace.py

```diff
@@ -8,10 +8,10 @@
 from pathlib import Path
 from typing import IO, AnyStr, Dict, Optional, Union
 
-from azure.ai.ml._restclient.v2023_02_01_preview.models import ManagedServiceIdentity as RestManagedServiceIdentity
-from azure.ai.ml._restclient.v2023_02_01_preview.models import FeatureStoreSettings as RestFeatureStoreSettings
-from azure.ai.ml._restclient.v2023_02_01_preview.models import Workspace as RestWorkspace
-from azure.ai.ml._restclient.v2023_02_01_preview.models import ManagedNetworkSettings as RestManagedNetwork
+from azure.ai.ml._restclient.v2022_12_01_preview.models import ManagedServiceIdentity as RestManagedServiceIdentity
```

Comment:
@joshharrin As v2023-02-01_preview doesn't not have few of these APIs reverting to v2022-12.
Please help review and let me know if I miss any changes correspond to [this PR](https://github.com/Azure/azure-sdk-for-python/pull/28879)

Reply:
looks good now